akka-docs/_sphinx/themes/akka/layout.html

```diff
@@ -217,10 +217,13 @@
   docsearch({
     apiKey: '543bad5ad786495d9ccd445ed34ed082',
     indexName: 'akka_io',
-    inputSelector: '#q',
+    inputSelector: '#search',
     algoliaOptions: {
       hitsPerPage: 10,
-      facetFilters: '["language:' + lang + '","version:' + version + '"]'
+      facetFilters: '[' +
+      '"language:' + lang + '"' +
+      '"language:general"' +
```

Comment:
dang, missed a `,`, but yes. Sorry, I did not CC entire team on algolia mailings on this one – they've added additional indexing now basically.

```
I've added the "general" pages. It's in the facet "language". So to search on Scala or General page, and a specific VERSION just write: 
facetFilters: [
  ["language:scala","language:general"],
  "version:VERSION"
]
```